code-scanning/apisec-scan.yml

```diff
@@ -10,7 +10,7 @@
 # How to Get Started with APIsec.ai
 # 1. Schedule a demo at https://www.apisec.ai/request-a-demo .
 #
-# 2. Register your account at https://cloud.fxlabs.io/#/signup .
+# 2. Register your account at https://cloud.apisec.ai/#/signup .
```

Comment:
Would need to verify your identity before doc changes. 
@mattlovestech Can you please do that?

Reply:
@Phantsure @mattlovestech it has been a week gone to marge my small commit. Please approve it ASAP as we have pending work on it

Reply:
T And also I'm sending this from my phone iPhone because my laptop is hacked. I sent this from my iPhone because I took a picture from my laptop phone Google Chrome. hese people are hacking me I don't care what you do just do whatever you have to do it because these people and would not stop hacking me over and over. I will send more pictures about these people look at in this area. I'll try many time to call taiga, CIA and this is again this is the location and also they hacking the gig hubs abs many times to prevent me.![image](https://user-images.githubusercontent.com/41128366/212981446-987ac1e4-67f8-40b0-bcf3-8e61e4712607.png)![image](https://user-images.githubusercontent.com/41128366/212981471-752db945-2d3d-488b-8b5b-c7f4dd5383c8.jpeg)

Reply:
Take a look there, hockey prevent me to send information![image](https://user-images.githubusercontent.com/41128366/212981710-93bd6212-cac4-46b1-9703-12fe694b7677.png)

Reply:
Lilliam Collado Gutierrez this my name 07/24/1977 my birthday

Reply:
@Phantsure it does not appear that API Sec is apart of the Technology Partnerships Program.

Reply:
@Phantsure @mattlovestech What we need to do now APISec is already a part of TPP

Reply:
> Prior to submitting a new workflow, please apply to join the GitHub Technology Partner Program: [partner.github.com/apply](https://partner.github.com/apply?partnershipType=Technology+Partner).

@mohdrashid1 This might help ⬆️

Reply:
@Phantsure We have apply for GitHub Technology Partner Program. Please update us and approve PR

Reply:
@mattlovestech Can you verify if they meet requirements?

Reply:
@mattlovestech @Phantsure r u guys done with checking our(apisec.ai) GTTP requirements.

Reply:
@mohdrashid1 can you send an email to partnerships@github.com to confirm that this PR was submitted by the APIsec team? After we get confirmation this can be approved.

Reply:
@mattlovestech   We sent an email from syedimran@apisec.ai email address to partnerships@github.com to confirm that this PR #1898 was by APIsec team.
Please review it and aprprove this PR.

Reply:
@Phantsure confirming an email was sent, authors are from apisec.ai